source/chapters/user_interface.rst

```diff
@@ -605,29 +606,55 @@ Headphone and Master Mix Controls
   .. note:: Don't forget to activate the :guilabel:`PFL` button on the deck you
             want to listen to in your headphones.
 
-**Head volume Knob**
-  This button adjusts your headphone volume. You can adjust the volume of a
-  single deck's signal you are listening to in the headphones with the
-  :ref:`Gain <interface-eq-gain>` knob.
-
 **Head-Split Button**
-  If activated, two mono signals are send to the headphone output instead of one
+  If activated, two mono signals are sent to the headphone output instead of one
   stereo signal. The master signal plays in the right channel, while the cueing
-  signal plays in the left channel. You can still adjust the head-mix so in the
-  left channel is not the pure cueing signal.
+  signal plays in the left channel. You can still adjust the
+  :guilabel:`Head-Mix Knob` to control the mix of the master and cue signals in
+  the left channel.
 
 **Balance Knob**
   This knob allows you to adjust the :term:`balance` (left/right orientation) of
   the master output.
 
-**Volume Knob**
-  The Volume Knob controls the overall volume of the master output. Adjust
-  this knob so that the :ref:`Master VU meters <interface-vu-meter>` are just
-  at the peak.
-
-  .. hint:: If the Peak indicator on top of the Master VU meter flashes, the
-            master output signal is clipping (too loud). Lower the volume with
-            using the volume knob.
+.. _interface-gain-knob:
+  
+**Gain Knob**
+  The Gain Knob controls the gain of the signal that Mixxx sends to your
+  sound card. Generally, this knob should not be adjusted. To change the output
```

Comment:
We had a lengthy "headroom" discussion about it when reworking the normalisation preferences. 

With the current default setting and an external volume knob in place, you are fully right, master gain should not be touched. 

But it seams that there are users who prefer not to use ReplayGain and play the track at 0 dB.
Since most pop tracks are levelled to full digital scale, they will mediately suffer clipping once they start to mix or  use the EQs or whatever. These users can reduce master gain to heave a headroom for mixing.
(IMHO they are doing wrong, since the Mixxx engine does not expect such high level tracks.)